example/example12.cpp

```diff
@@ -52,7 +52,7 @@ class PyExample12 : public Example12 {
         PYBIND11_OVERLOAD_PURE(
             bool,
             Example12,
-            run_bool
+            run_bool,
```

Comment:
Agree, this looks weird. I'm sure there is a way to fix a macro instead, I'll have a look, but it might be more trouble than it's worth, working with preprocessor is always a pain.

I thinks warnings like this in example code are fine, since they don't affect library users.